telegramManager.js

```diff
@@ -0,0 +1,20 @@
+const axios = require('axios');
+const config = require('./config');
+
+const getChatAdministrators = (chatId) => {
+  return axios.post(`${config.telegramBaseUrl}/getChatAdministrators`, {
+    chat_id: chatId,
+  });
+};
+
+const talkToBot = (chatId, text) => {
+  return axios.post(`${config.telegramBaseUrl}/sendMessage`, {
+    chat_id: chatId,
+    text: text
```

Comment:
http://www.quote-coyote.com/album/small/To-be-or-not-to-be-that-is-the-question.jpg